scripts/upload_version.sh

```diff
@@ -12,18 +12,14 @@
 # Usage:    Should be used after build_packages.sh completes successfully
 #           Currently called by deploy_version.sh
 #
-# Examples: scripts/upload_version.sh nightly $HOME/node_pkg/
+# Examples: scripts/upload_version.sh nightly $HOME/node_pkg/ algorand-releases
 
 if [[ "$#" -ne 2 && "$#" -ne 3 ]]; then
-    echo "Syntax: upload_version <channel> <directory-with-packages> <optional: bucket>"
+    echo "Syntax: upload_version <channel> <directory-with-packages> <bucket>"
```

Comment:
The if clause above is no longer correct. the number of argument need to be exactly 3.

Reply:
done